src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -149,21 +154,20 @@ public boolean containsAny(Prefix... prefixes) {
     }
 
     /**
-     * Checks if every single one of the prefixes exists in the map
+     * Checks if the preamble of the argumentMultimap object is empty
      *
-     * @param prefixes a list of prefixes
-     * @return true if all the prefixes exists in the map
+     * @return true if the preamble is empty
      */
-    public boolean containsAll(Prefix... prefixes) {
-        return Stream.of(prefixes).allMatch(this::contains);
+    public boolean isPreambleEmpty() {
+        return this.getPreamble().isEmpty();
     }
 
     /**
-     * Checks if the preamble of the argumentMultimap object is empty
+     * Checks if the preamble of ArgumentMultimap object is made of 1 part
      *
-     * @return true if the preamble is empty`
+     * @return true if the preamble is by itself
      */
-    public boolean isPreambleEmpty() {
-        return this.getPreamble().isEmpty();
+    public boolean isPreambleAlone() {
```

Comment:
The name `isPreambleAlone` gives me the impression that there is only preamble in the command. e.g. `edit 1 2 3 4 5` would return `true` as there is are no prefixes, only the command word and preamble.

Perhaps a more suitable name could be `hasOnlyOnePreambleSegment`?

Reply:
thanks for the sugestion